docs/app/Examples/modules/Dropdown/Content/AsyncOptions.js

```diff
@@ -29,29 +28,41 @@ export default class DropdownAsyncOptions extends Component {
 
   selectRandom = () => this.setState({ value: _.sample(this.state.options).value })
   toggleSearch = (e) => this.setState({ search: e.target.checked })
+  toggleMultiple = (e) => {
+    const { value } = this.state
+    // convert the value to/from an array
+    const newValue = e.target.checked ? _.compact([value]) : value[0]
+    this.setState({ multiple: e.target.checked, value: newValue })
+  }
```

Comment:
Since this example allows toggling the `multiple` prop, we need to convert the value to and from an array corresponding to the dropdown type.